composer.json

```diff
@@ -17,7 +17,10 @@
         "hhvm/hhast": "^4.0",
         "facebook/definition-finder": "^2.0.0",
         "hhvm/hacktest": "^2.0",
-        "usox/hackmock": "dev-master",
         "facebook/fbexpect": "^2.6.1"
+    },
+    "scripts": {
+        "post-install-cmd": "patch --dry-run -R -p1 -i hackttp.patch || patch -p1 -i hackttp.patch",
+        "post-update-cmd": "patch --dry-run -R -p1 -i hackttp.patch || patch -p1 -i hackttp.patch"
```

Comment:
first part succeeds if the patch is already applied and causes the second part to be skipped